### PR TITLE
Add niconico

### PR DESCRIPTION
**Pull request by sckemu**
[Niconico](https://www.nicovideo.jp/) is a popular video site in Japan ([Alexa Rank 107](https://www.alexa.com/siteinfo/nicovideo.jp)).
However, I could not find an English TFA setup steps.
Japanese: https://qa.nicovideo.jp/faq/show/4500

### CODE REVIEW

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
I compressed the logo 60%:
niconico
Other than that the PR looks good 👍

**Comment by Carlgo11** [comment]
On a side note, is it just me or does Japanese sites seem to prefer low quality fonts?
They look like Arial in a really lossy compression of JPEG. 🤔

**Comment by sckemu** [comment]
It may be because I'm looking at a browser with Japanese settings, but I can see this page without any problems.

**Review by Carlgo11 — DISMISSED** [review]

File: _data/entertainment.yml [hunk 1]
```diff
@@ -86,6 +86,13 @@ websites:
       img: netflix.png
       tfa: No
 
+    - name: Niconico
+      url: https://www.nicovideo.jp/
+      img: niconico.png
+      tfa: Yes
+      software: Yes
+      doc: https://qa.nicovideo.jp/faq/show/4500
+
```
Comment (kmpoppe) [inline _data/entertainment.yml]: `     lang: jp`

Comment (kmpoppe) [inline _data/entertainment.yml]: I think the language could be included.

Comment (kmpoppe) [reply _data/entertainment.yml]:
```suggestion
lang: jp
```

Comment (sckemu) [reply _data/entertainment.yml]: "ja" instead of "jp" is correct.

**Review by kmpoppe — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**